alarm/utils.py

```diff
@@ -33,3 +30,128 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def refresh_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in Threshold.objects.filter(trade_pair=trade_pair):
```

Comment:
Why do not use threshold from the line above?

Reply:
++